src/DevTools/Extension/Extension.tsx

```diff
@@ -34,10 +40,9 @@ const ShellTriggerButton = React.forwardRef<HTMLButtonElement>((_, ref) => {
       variant="filled"
       color={useThemeMode('dark', 'gray.3')}
       onClick={() => setIsShellOpen(true)}
-      sx={shellTriggerButtonStyles}
       ref={ref}
       title="Open Jotai Devtools"
-      className="jotai-devtools-trigger-button"
+      className={shellTriggerButtonClassName}
```

Comment:
Thank you for the PR! 🙏

Do you think we should not render the button at all if it's SSR? 🤔 Also, do we see similar issue when rendering `<DevTools isInitialOpen />`?

Reply:
From what I can see, isShellOpenAtom will always be null on the server, so this would cover that case regardless.

Actually, yeah it's probably better to just not render it in SSR. 
I tested a few things, it seems if we don't return the button html on the server we get a hydration mismatch error.
The only working solution I found was this: https://nextjs.org/docs/messages/react-hydration-error#solution-1-using-useeffect-to-run-on-the-client-only

Not sure how you feel about adding that to `Extension`?

Another alternative (Next 13 specific) is to instruct the user to create their own wrapper using `dynamic`:

```tsx
import { DevTools } from "jotai-devtools";
import dynamic from "next/dynamic";
import React from "react";

const JotaiDevTools: typeof DevTools = (props) => {
  return <DevTools {...props} />;
};

export default dynamic(() => Promise.resolve(JotaiDevTools), { ssr: false });
```

Reply:
Sorry for the late response! 

Yeah, not a huge fan of adding "next/dynamic" as a dependency for this use case. I wonder if this is something our consumers could do on their end. 🤔  Do you think we should add it to the guide along with your CSS fix?

Reply:
Oh for sure, the next/dynamic part shouldn't be part of this pkg.
TBH I think this CSS fix is enough to make the issue no longer be a concern.
So documenting the other possible fixes wouldn't really be necessary if this is merged. I guess it wouldn't hurt though?

Reply:
That makes sense! I'll merge this soon + update dependencies and cut a release. Thanks again for your patience and your contribution 🙏